mautic/mautic
# Fix an issue where dateFrom and dateTo are null in the 'Contact Segment Activity Log' report

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ &check; ]
| New feature/enhancement? (use the a.x branch)      | [&cross;]
| Deprecations?                          | [&cross;]
| BC breaks? (use the c.x branch)        | [&cross;]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:


An error is returned when trying to retrieve the `Contact's segments activity log` report via the API without parameters. This PR fixes it.
#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create the report with `Contact's segments activity log` as a data source.
3. Try to get the report via API `/api/reports/ID`.
4. It works well. 500er was before
